sci-libs/openblas/openblas-0.2.15.ebuild

```diff
@@ -197,4 +197,7 @@ src_install() {
 	multilib_install_wrappers
 
 	dodoc GotoBLAS_{01Readme,03FAQ,04FAQ,05LargePage,06WeirdPerformance}.txt *md Changelog.txt
+
+	# fix cmake installation
+	sed -i -e 's@'"${ED}"'@/@g' ${ED}/usr/$(get_libdir)/cmake/openblas/OpenBLASConfig.cmake || die
```

Comment:
Here is the version I'm using currently https://github.com/Heather/gentoo-cynede/blob/master/sci-libs/openblas/openblas-0.2.19.ebuild
I have some TODO problems with it:
 - with alternatives eclass, why does it change custom eselect apps to single eselect?
 - This ebuild also must create symlink to libopenblas.so (from libopenblas_threaded.so)

when I will have time to resolve those I will try to provide final version
what a nicer way for sed expression?